subprojects/docs/src/docs/userguide/dependencyManagement.adoc

```diff
@@ -198,102 +198,149 @@ Let's say you wanted to declare a dependency on the link:https://tomcat.apache.o
-=== Customizing dependency declarations
+[[sub:resolve_specific_artifacts_from_dependency]]
+==== Resolving specific artifacts from a module dependency
 
-[[ssub:multi_artifact_dependencies]]
-==== Depending on modules with multiple artifacts
+Whenever Gradle tries to resolve a dependency from a Maven or Ivy repository, it looks for a metadata file and the default artifact file, a JAR. The build fails if none of these artifact files can be resolved. Under certain conditions, you might want to tweak the way Gradle resolves artifacts for a dependency.
 
-As mentioned earlier, a Maven module has only one artifact. Hence, when your project depends on a Maven module, it's obvious what its artifact is.
-With Gradle or Ivy, the case is different. Ivy's dependency descriptor (`ivy.xml`) can declare multiple artifacts.
-For more information, see the Ivy reference for `ivy.xml`.
-In Gradle, when you declare a dependency on an Ivy module, you actually declare a dependency on the `default` configuration of that module.
-So the actual set of artifacts (typically jars) you depend on is the set of artifacts that are associated with the
-`default` configuration of that module. Here are some situations where this matters:
+- The dependency only provides a non-standard artifact without any metadata e.g. a ZIP file.
+- The dependency metadata declares more than one artifact e.g. as part of an Ivy dependency descriptor.
+- You only want to download a specific artifact without any of the transitive dependencies declared in the metadata.
 
-* The `default` configuration of a module contains undesired artifacts. Rather than depending on the whole configuration, a dependency on just the desired artifacts is declared.
-* The desired artifact belongs to a configuration other than `default`. That configuration is explicitly named as part of the dependency declaration.
+Gradle is a polyglot build tool and not limited to just resolving Java libraries. Let's assume you wanted to build a web application using JavaScript as the client technology. Most projects check in external JavaScript libraries into version control. An external JavaScript library is no different than a reusable Java library so why not download it from a repository instead?
 
-There are other situations where it is necessary to fine-tune dependency declarations.
-Please see the api:org.gradle.api.artifacts.dsl.DependencyHandler[] class in the API documentation for examples and a complete reference for declaring dependencies.
+link:https://developers.google.com/speed/libraries/[Google Hosted Libraries] is a distribution platform for popular, open-source JavaScript libraries. With the help of the artifact-only notation you can download a JavaScript library file e.g. JQuery. The `@` character separates the dependency's coordinates from the artifact's file extension. The api:org.gradle.api.artifacts.dsl.DependencyHandler[] also allows for declaring the `extension` attribute as map notation.
```

Comment:
OK, I will remove it for now.

Reply:
Since this documentation mainly targets new users in my mind, I think removing this here is fine. There are a lot of details and additional functionality which is not documented in this section. If I am a more advanced user and already know about map notation, I can still find information on it (e.g. in the API docs of DependencyHandler).